Palaso.Tests/WritingSystems/EthnologueLookupTests.cs

```diff
@@ -140,5 +137,24 @@ public void SuggestLanguages_NonPrimary_ResultIncludesAlternateLanguageNames()
 			Assert.True(languages.Any(l => l.Names.Contains("Andaki")));
 			Assert.True(languages.Any(l => l.Names.Contains("Churuba")));
 		}
+
+		[Test]
+		public void SuggestLanguages_Akan_DoesnotCrash()
+		{
+			var languages = _ethnologue.SuggestLanguages("a");
+			Assert.True(languages.Any(l => l.Code == "ak"));
+			Assert.True(languages.Any(l => l.Code == "akq"));
+			Assert.True(languages.Any(l => l.Names.Contains("Akuapem")));
+			Assert.True(languages.Any(l => l.Names.Contains("Ak")));
+			Assert.True(languages.Any(l => l.Names.Contains("Akan")));
+			Assert.True(languages.Any(l => l.Names.Contains("Fanti")));
+			languages = _ethnologue.SuggestLanguages("ak");
+			Assert.True(languages.Any(l => l.Code == "ak"));
+			Assert.True(languages.Any(l => l.Code == "akq"));
+			Assert.True(languages.Any(l => l.Names.Contains("Asante")));
+			Assert.True(languages.Any(l => l.Names.Contains("Ak")));
+			Assert.True(languages.Any(l => l.Names.Contains("Akan")));
+			Assert.True(languages.Any(l => l.Names.Contains("Fanti")));
+		}
 	}
```

Comment:
Generally, I prefer to include a human understandable message for assertion failures.  But I wouldn't hold up a checkin just for that...